schemathesis_hooks.py

```
@@ -0,0 +1,164 @@
+                            users_name.add(name)
+                            continue
+                        if field.name.value == "email":
+                            email = field.value.value
+                            if email == "":
+                                return False
+                            if email in users_email:
+                                return False
+                            users_email.add(email)
+                            continue
+        return True
+    except AttributeError as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)